scripts/get-hashsets.js

```diff
@@ -50,6 +67,20 @@ function handleBreachesResponse(error, response, body) {
   try {
     const breachesJSON = JSON.parse(body);
 
+    s3.listObjects({Bucket: BREACH_HASHSET_BUCKET_NAME}, (err, data) => {
+      if (err) {
+        console.error(`listObjects, err: ${err}`);
+      } else {
+        console.log(`listObjects, data: ${data}`);
+      }
+    });
+    s3.createBucket({Bucket: BREACH_HASHSET_BUCKET_NAME}, (err, data) => {
+      if (err) {
+        console.error(`createBucket, err: ${err}`);
+      } else {
+        console.log(`createBucket, data: ${data}`);
+      }
+    });
```

Comment:
Nit: these two methods, `s3.listObjects()` and `s3.createBucket()`, should probably use the same syntax as `s3.upload()` above on L49-53.